### PR TITLE
fix op to rc in comment of FieldContext.Child

### PR DESCRIPTION
**Pull request by s-ichikawa**
Fixed the implementation in the comments.

I have:
 - [ ] Added tests covering the bug / feature (see [testing](https://github.com/99designs/gqlgen/blob/master/TESTING.md))
 - [ ] Updated any relevant documentation (see [docs](https://github.com/99designs/gqlgen/tree/master/docs/content))

### CODE REVIEW

File: graphql/context_field.go [hunk 1]
```diff
@@ -36,8 +36,8 @@ type FieldContext struct {
 	//
 	//	srv.AroundFields(func(ctx context.Context, next graphql.Resolver) (interface{}, error) {
 	//		fc := graphql.GetFieldContext(ctx)
-	//		op := graphql.GetOperationContext(ctx)
-	//		collected := graphql.CollectFields(opCtx, fc.Field.Selections, []string{"User"})
+	//		rc := graphql.GetOperationContext(ctx)
```
Comment (s-ichikawa) [inline graphql/context_field.go:39]: Actually, there are some variations in the naming of the variable returned from `graphql.GetOperationContext()` (e.g., `rc`, `opCtx`, `rctx`), but `rc` is the most common.

**Comment by StevenACoffman** [comment]
Hey, I appreciate making things moe consistent, but it feels unintuitive for the `graphql.GetOperationContext(ctx)` to be assigned to `rc`. If we have a mix, I would rather find the places we are currently using `rc` for this and change them to `opCtx`.

**Comment by s-ichikawa** [comment]
@StevenACoffman maybe I changed them all.

**Comment by StevenACoffman** [comment]
Thanks so much! This has confused a number of people, so I appreciate you cleaning it up! Looking forward to future contributions from you!

**Pull request merged**